python-backend/app/api/mines/mine_api_models.py

```diff
@@ -251,51 +257,68 @@ def format(self, value):
     })
 
 MINE_STATUS_CODE_MODEL = api.model(
-        'MineStatusCode', {
-            'mine_status_xref_guid':fields.String(),
-            'mine_operation_status':fields.Nested(MINE_OPERATION_STATUS_CODE_MODEL),
-            'mine_operation_status_reason':fields.Nested(MINE_OPERATION_STATUS_REASON_CODE_MODEL),
-            'mine_operation_status_sub_reason':fields.Nested(MINE_OPERATION_STATUS_SUB_REASON_CODE_MODEL),
-            'description': fields.String(),
+    'MineStatusCode', {
+        'mine_status_xref_guid': fields.String(),
```

Comment:
There's a mix of `fields.String()` and `fields.String`. I've only used `fields.String`. Does invoking the method do anything different?

Reply:
Oh, not sure - this just came out of my pep8 formatting. You're probably right that it should consistently be `String` - but I haven't seen any issues.

Reply:
I'll switch over and see what the postman looks like